grpc/src/main/proto/language_id.proto

```diff
@@ -14,7 +14,22 @@ message LanguageIdResponse {
   string langId = 1;
 }
 
+message IdResult {
+  string id = 1;
+  double score = 2;
+}
+
+message LanguageIdScoresResponse{
+  repeated IdResult idResult = 1;
+}
+
 service LanguageIdService {
   rpc Detect (LanguageIdRequest) returns (LanguageIdResponse);
   rpc DetectFast (LanguageIdRequest) returns (LanguageIdResponse);
+  rpc getScores (LanguageIdRequest) returns (LanguageIdScoresResponse);
```

Comment:
I would prefer removing some of those methods and adding boolean flags to the LanguageIdRequest. Such as bool fast and bool useTrGroup. So result item would be the same. Even you can put "bool include scores" and put repeated IdResult idResult = 1; to the LanguageIdRespose. What do you think?

Reply:
Fixed as u advise.